webcompat/static/js/lib/bugform.js

```diff
@@ -155,6 +156,18 @@ function BugForm() {
     img.src = dataURI;
   };
 
+  // Event handler for image file drop; purpose to detect multiple drops
+  this.handleImageDrop = function() {
+    var imageFilled = false;
```

Comment:
We probably don't need this now, since we have the `hasImage` property on `BugForm` now.